invenio_demosite/base/fixtures/websubmit.py

```diff
@@ -17,117 +17,164 @@
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
+"""Collection fixtures."""
+
 import datetime
+
 from fixture import DataSet
 
 
 class SbmCOLLECTIONData(DataSet):
 
-    class SbmCOLLECTION_36:
-        id = 36L
+    """SbmCOLLECTIONData."""
+
+    class SbmCOLLECTION_1:
+        id = 1L
         name = u'Document Types'
 
 
 class SbmCOLLECTIONSbmCOLLECTIONData(DataSet):
 
+    """SbmCOLLECTIONSbmCOLLECTIONData."""
+
     class SbmCOLLECTIONSbmCOLLECTION_0_36:
-        id_son = SbmCOLLECTIONData.SbmCOLLECTION_36.ref('id')
+        son = SbmCOLLECTIONData.SbmCOLLECTION_1
         catalogue_order = 1L
         id_father = None
 
 
 class SbmDOCTYPEData(DataSet):
 
+    """SbmDOCTYPEData."""
+
     class SbmDOCTYPE_DEMOART:
         md = datetime.date(2008, 3, 6)
         ldocname = u'Demo Article Submission'
-        description = u'<br /><br />The Demo Article submission demonstrates a more complex submission type.<br /><br />\r\nThe submission gives a document a category. This category is used in the document\'s reference number and also serves as a means to classify it into a specific ATLANTIS collection. Documents submitted into the "Article" category are inserted into the ATLANTIS "Articles" collection, documents categorized as "Preprint" are inserted into the ATLANTIS "Preprints" collection, and a document categorized as a "Report" is inserted into the ATLANTIS "Reports" collection.<br /><br />\r\n'
+        description = u'<br /><br />The Demo Article submission demonstrates ' + \
```

Comment:
please use following syntax for breaking a string line:

```python
foo = ('aaa '
       'bbb')
# or 
foo = 'aaa' \
      'bbb'
```

--
http://stackoverflow.com/questions/1874592/pep8-80-characters-big-strings